public/js/controller-stats.js

```diff
@@ -86,7 +86,7 @@ function StatsCtrl($scope, $rootScope, $route, $http, socket, resolvedData) {
   };
   Player.prototype.totaldamage = function() {
     var arr = [];
-    for (var i = 0, len = Math.max(this.damagedone.length, this.overkillDamage.length); i < len; i++) {
+    for (var i = 0, len = Math.min(this.damagedone.length, this.overkillDamage.length); i < len; i++) {
```

Comment:
Using Math.min causes the array to be empty, when overkill damage doesn't exist (old version of the plugin), which causes the table to display "-" instead of the damage number.